docs/examples/index.rst

```diff
@@ -0,0 +1,118 @@
+*********
+Examples
```

Comment:
I don't find the left hand side navigation intuitive: "Guides", "Advanced Topics", and "API Reference" don't have an actual content, but are merely "index pages" that mirror the content from the navigation (i.e. list child pages).

"Examples" on the other hand is very different, it shows most of the important information.

I would suggest that you make do NOT list the actual examples in the navigation. Additionally, I think it would be better to have "Examples" without actual content like the other expandable sections, and move the content of "index.rst" into a separate file.

Reply:
@andsteing  I've updated navigation + page content to address the above.